src/go/k8s/README.md

```diff
@@ -18,6 +18,7 @@ Official Kubernetes quick start documentation can be found at
 * Kubernetes 1.16 or newer
 * kubectl 1.16 or newer
 * kustomize v3.8.7 or newer
+* cert-manager v1.2.0 or newer
```

Comment:
I think we're fine with 1.0.0+ - I can add test for that. I think this is pretty hard requirement since that got released like 2 weeks ago and it takes time for people to install

Reply:
Ok, changed.